system/jlib/jcontainerized.cpp

```diff
@@ -267,7 +271,7 @@ bool applyYaml(const char *componentName, const char *wuid, const char *job, con
     if (autoCleanup)
     {
         unsigned deleteJobGracePeriod = 0;
-        if (strcmp(resourceType, "job") == 0)
+        if (streq(resourceType, "job") == 0)
```

Comment:
not really related to PR, tidying up..
Unfortunately change is wrong. (i.e. streq is a bool func.)